plugins/katello/3.13/installation/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: plugins/katello/documentation
+title: Katello Installation
+version: 3.13
+foreman_version: 1.23
+latest: 3.13
+script: osmenu.js
+---
+
+# Katello {{ page.version }} Installation
+
+{% if page.version == '3.13' %}
```

Comment:
I think the sed replace you used for 'nightly' was too aggressive - this should say nightly instead of 3.13